File: aifupan_project/ReviewAnalysis/vo/AnalysisResultVo.cs
using ReviewAnalysis.Asr;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ReviewAnalysis.vo
{
    public class AnalysisResultVo
    {
        /// <summary>
        /// 段落分析信息
        /// </summary>
        public List<SentenceMarkVo> sentenceMarkVos { get; set; }

        /// <summary>
        /// 分析结果的关键词类型信息
        /// </summary>
        public List<AnalysisResultCruxTypeVo> cruxTypeList { get; set; }

        /// <summary>
        /// 词语汇总列表
        /// </summary>
        public List<WordsMarkVo> wordsCollect { get; set; }

        /// <summary>
        /// 词语tab列表
        /// </summary>
        public List<WordsTabVo> wordsTabList { get; set; }
    }
}

File: aifupan_project/fupan-server/replay-generic/src/main/java/com/jiuyu/replay/generic/vo/words/AnalysisResultVo.java
package com.jiuyu.replay.generic.vo.words;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

import java.util.List;

@Data
@Schema(description = "分析结果")
public class AnalysisResultVo {

    /**
     * 段落分析信息
     */
    @Schema(description = "段落分析信息")
    private List<SentenceMarkVo> sentenceMarkVos;
    /**
     * 分析结果的关键词类型信息
     */
    @Schema(description = "分析结果的关键词类型信息")
    private List<AnalysisResultCruxTypeVo> cruxTypeList;
    /**
     * 词语汇总列表
     */
    @Schema(description = "词语汇总列表")
    private List<WordsMarkVo> wordsCollect;
    /**
     * 词语tab列表
     */
    @Schema(description = "词语tab列表")
    private List<WordsTabVo> wordsTabList;
}
